rero_ils/modules/libraries/api_views.py

```diff
@@ -79,3 +86,78 @@ def list_closed_dates(library_pid):
         },
         'closed_dates': closed_date
     })
+
+
+@api_blueprint.route('/<library_pid>/statistics/circulation', methods=['GET'])
```

Comment:
Using a `ContentNegotiatedMethodView` from `invenio_rest` can be a better approach. See `ImportsListResource` as example.